src/all/unionmangas/assets/i18n/messages_it.properties

```diff
@@ -0,0 +1,5 @@
+chapters_cannot_be_processed=I capitoli non possono essere elaborati
```

Comment:
is this mtl? please remove it so

edit: intl not even needed see my other review on the exception throwing